test/functional/executor.py

```diff
@@ -56,6 +58,10 @@ def run(self):
         passed = 0
         for test in self.tests:
             cmd = [self.executable, *test.get_command()]
+            if self.verbose:
+                print("TEST: ", test.get_name())
+                print("DESCRIPTION: ", test.get_description())
+                print("RUNNING: ", *cmd)
             subprocess.run(cmd, stdout=subprocess.DEVNULL,
                            stderr=subprocess.DEVNULL)
```

Comment:
But there is no possibility to output anything from the application, because `stdout` and `stderr` are redirected to `/dev/null` in every case.